docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,398 @@
+roles in {es}, {kib}, {ls}, and {metricbeat}. 
+
+[float]
+[[gs-security-prereqs]]
+=== Before you begin
+
+. Install and configure {es}, {kib}, {ls}, and {metricbeat} as described in 
+<<get-started-elastic-stack>>. 
+
+. Stop {ls}. The method for starting and stopping {ls} varies depending on whether 
+you are running it from the command line or running it as a service. For example, 
+if you are running {ls} from the command line, you can stop it by entering 
+`Ctrl-C`. See {logstash-ref}/shutdown.html[Shutting down {ls}]. 
+
+. Stop {metricbeat}. For example, enter `Ctrl-C` on the command line where it is 
+running.
+
+. Launch the {kib} web interface by pointing your browser to port 5601. For 
+example, http://127.0.0.1:5601[http://127.0.0.1:5601].
+
+[float]
+[[gs-security-license]]
+=== Install a trial license
+
+include::{docdir}/getting-started/get-started-trial.asciidoc[]
+
+[float]
+[[gs-security-enabled]]
+=== Enable security in {es}
+
+include::get-started-enable-security.asciidoc[]
+
+NOTE: This tutorial involves a single node cluster, but if you had multiple 
+nodes, you would enable {security} on every node in the cluster and configure 
+Transport Layer Security (TLS) for internode-communication, which is beyond the 
+scope of this tutorial. 
+
+[float]
+[[gs-built-in-users]]
+=== Create passwords for built-in users
+
+include::get-started-builtin-users.asciidoc[]
+
+You need these built-in users in subsequent steps, so choose passwords that you 
+can remember!
+
+NOTE: This tutorial does not use the built-in `logstash_system` and 
+`beats_system` users, which are typically associated with monitoring. For more 
+information, see 
+{logstash-ref}/ls-security.html#ls-monitoring-user[Configuring credentials for {ls} monitoring]
+and {metricbeat-ref}/monitoring.html[Monitoring {metricbeat}].  
+  
+[float]
+[[gs-kibana-security]]
+=== Add the built-in user to {kib}
+
+When {security} is enabled, users must log in to {kib} with a valid user ID and 
+password. 
+
+{kib} also performs some tasks under the covers that require use of the 
+built-in `kibana` user. 
+
+. If you don't mind having passwords visible in your configuration file, 
+uncomment and update the following settings in the `kibana.yml` file in your 
+{kib} directory:
++
+--
+TIP: If you installed {kib} using archive distributions (`zip` or 
+`tar.gz`), the `kibana.yml` configuration file is in `KIBANA_HOME/config`. If 
+you used package distributions (Debian or RPM), it's in `/etc/kibana`. For more 
+information, see {kibana-ref}/settings.html[Configuring {kib}].  
+
+For example, add the following settings:
+
+[source,yaml]
+----
+elasticsearch.username: "kibana"
+elasticsearch.password: "your_password"
+----
+
+Specify the password that you set with the `elasticsearch-setup-passwords` 
+command then save your changes to the file. 
+--
+
+. If you prefer not to put your user ID and password in the `kibana.yml` file, 
```

Comment:
1 and 2 are mutually exclusive so I would not make them separate steps. Noticed this because I performed step 1 and then saw step 2...and thought...no, I want to do that instead!